### DISCUSSION TITLE
Advice on upgrading from Bouncy Castle FIPS 1.0.2.x to Bouncy Castle 2.0.

### DISCUSSION THREAD

**Discussion by joeburke30** [bcgit/bc-java#1850] [General]
I've been searching around the sites and documentation but haven't found my answer.   
We are looking to upgrade to the latest BC FIPS version. 
I need to assess if there are any breaking changes between the 1.0.2 and 2.0.0 series.   
Is there any resources where I can find this?

**Comment by dghgit** [comment]
You'll find a porting guide on https://www.bouncycastle.org/download/bouncy-castle-java-fips/ in the distribution block for BC-FJA 2.0.0.
